web/package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint --ext .js,.ts,.tsx, src"
+    "lint": "eslint --ext .js,.ts,.tsx, src --fix"
```

Comment:
We use this script to check if eslint has some warnings or errors instead of fixing them. How about add a new scripts like `lint-fix`?

Reply:
I think we should not change it in this PR. This PR is mainly introducing a new language. WDYT

Reply:
Right, I think the same way, that's why I've reinstated it again.